### PR TITLE
Add support for overriding the conversion type of manually implemente…

### PR DESCRIPTION
**Pull request by sdroege**
…d types

Fixes https://github.com/gtk-rs/gir/issues/476

### CODE REVIEW

**Comment by sdroege** [comment]
Usage of this can be seen here https://github.com/sdroege/gstreamer-rs/issues/3

File: src/config/gobjects.rs [hunk 1]
```diff
@@ -213,6 +231,10 @@ fn parse_object(toml_object: &Value, concurrency: library::Concurrency) -> GObje
         warn!("ref_mode configuration used for non-manual object {}", name);
     }
 
+    if status != GStatus::Manual && conversion_type != None {
```
Comment (EPashkin) [inline src/config/gobjects.rs]: `.is_none()` ?

File: src/config/gobjects.rs [hunk 2]
```diff
@@ -121,6 +123,17 @@ fn ref_mode_from_str(ref_mode: &str) -> Option<ref_mode::RefMode> {
     }
 }
 
+fn conversion_type_from_str(conversion_type: &str) -> Option<conversion_type::ConversionType> {
+    match conversion_type {
```
Comment (EPashkin) [inline src/config/gobjects.rs]: Maybe `use analysis::conversion_type::ConversionType::*;` added?

File: src/config/gobjects.rs (hunk 1 above)
Comment (sdroege) [reply src/config/gobjects.rs]: Done

File: src/config/gobjects.rs (hunk 2 above)
Comment (sdroege) [reply src/config/gobjects.rs]: Done

**Comment by EPashkin** [comment]
Thanks, squash please and I merge after CI.

**Comment by sdroege** [comment]
Done

**Comment by EPashkin** [comment]
@sdroege Thanks

**Pull request merged**